src/site/serverless/affordable-housing.liquid

```diff
@@ -256,7 +256,7 @@ Narrow your search with the filter options below or <a href="#results">scroll</a
             {%- for status in allStatuses -%}
               <p>
                 {%- capture badgeMod -%}
-                  {%- if status == "Call for Status" -%}
+                  {%- if status == "Call for Availability" or status == "Call for Status" -%}
```

Comment:
You should be able to remove this "Call for Status" condition given line 123 in housing.js

Reply:
Fixed (https://github.com/theunitedeffort/theunitedeffort.org/pull/611/commits/5d37c68f79fe608ae30e617017fe2b51c42b6bb9)